packages/vuetify/src/util/mask.ts

```diff
@@ -89,7 +97,7 @@ const maskValidates = (mask, char) => {
  *
  * @return {String}
  */
-export const maskText = (text, masked, dontFillMaskBlanks) => {
+export const maskText = (text: string | any[], masked: string | any[], dontFillMaskBlanks: boolean): string => {
```

Comment:
shouldn't it be `string | string[]`?
also maybe it would be a good idea to add maskable.js as well, just as a additional verification if everything is ok